src/gd.c

```diff
@@ -1713,8 +1726,26 @@ static void dashedSet (gdImagePtr im, int x, int y, int color,
 					   int *onP, int *dashStepP, int wid, int vert);
 
 /*
-	Function: gdImageDashedLine
-*/
+ * Function: gdImageDashedLine
+ *
+ * Draws a dashed line
+ *
+ * Deprecated:
+ *   This function has been deprecated as of GD 2.3.0.  Use an appropriate
+ *   combination of <gdImageLine> and <gdImageSetStyle> instead.
+ *
+ * Parameters:
+ *   im    - The image.
+ *   x1    - The x-coordinate of the start point.
+ *   y1    - The y-coordinate of the start point.
+ *   x2    - The x-coordinate of the end point.
+ *   y2    - The y-coordinate of the end point.
+ *   color - The color.
+ *
+ * See also:
+ *   - <gdImageLine>
+ *   - <gdImageSetStyle>
+ */
 BGD_DECLARE(void) gdImageDashedLine (gdImagePtr im, int x1, int y1, int x2, int y2, int color)
 {
 	int dx, dy, incr1, incr2, d, x, y, xend, yend, xdirflag, ydirflag;
```

Comment:
seems like we should be able to gut this function and make it call `gdImageLine` & `gdImageSetStyle` ?  it'd simplify, shrink the code (compiled & source), and be a stronger proof that this func is indeed fully redundant with those other two.